components/alertbanner/stories/alertbanner.stories.js

```diff
@@ -2,9 +2,11 @@ import { disableDefaultModes } from "@spectrum-css/preview/modes";
 import { isOpen } from "@spectrum-css/preview/types";
 import pkgJson from "../package.json";
 import { AlertBannerGroup } from "./alertbanner.test.js";
+import { ActionableOptionsTemplate, Template, TextOverflowTemplate } from "./template.js";
 
 /**
- * The alert banner show pressing and high-signal messages, such as system alerts. They're meant to be noticed and prompt users to take action.
+ * The alert banner shows pressing and high-signal messages, such as system alerts. They're meant to be noticed and prompt users to take action.
+ * It will occupy all the available horizontal space until it reaches its maximum allowed width.
```

Comment:
Yeah, I'm not sure about us having that `max-content` on the Storybook root and that it could be making some components display not entirely accurately. Could this be throwing off the testing of max-widths? I also noticed it breaking a dynamic grid layout in a docs template that I wanted to expand and be responsive (I had to use fixed width columns instead). I believe that came along with the big modes update --- @castastrophe is this totally necessary for our Chromatic testing?

On a related note here is that in the Chromatic Variants testing grid, the examples are shrunk down too, but because they are a flex item within the testing grid's flex layout that doesn't take up the full horizontal space by default. I ended up adding the following to the docs only template, which is why you see them taking up more horizontal space (up until its max-width). I'm not sure if there's something to be done about that globally, but it's something to keep an eye out for: 
```
containerStyles: { alignItems: "stretch" },
wrapperStyles: { alignItems: "stretch" },
```
(these use the `column` direction so `alignItems` here affects the horizontal)

Reply:
Just a note for posterity that I've created a separate Jira issue **CSS-959** to us to look at that Storybook root element style.